tcms/templates/include/bugs_table.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+{% load static %}
+
+<div class="col-xs-12 col-sm-6 col-md-4">
+    <div class="card-pf card-pf-accented">
+        <h2 class="card-pf-title">
+            <span class="fa fa-bug"></span>
+            {% trans heading %}
```

Comment:
Translation should be outside of the inclusion, where you have the hard-coded string value. This should be display only.

Reply:
we can't do this because then this line - https://github.com/kiwitcms/Kiwi/pull/1718/files#diff-caa7fa8e502110e7b4c838750263a889R132
will become:
```
        {% include "include/bugs_table.html" with heading={% trans"Bugs" %} class="bugs" %}
```
which is not valid django template (Django does not allow for nested expressions)

Reply:
https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#trans-template-tag